packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -110,7 +110,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
           aria-live={ disabled || isOpen ? 'off' : 'assertive' }
           aria-label={ ariaLabel || label }
           aria-describedby={ id + '-option' }
-          aria-activedescendant={ selectedIndex >= 0 ? (this._id + '-list' + selectedIndex) : (this._id + '-list') }
+          aria-activedescendant={ isOpen && selectedIndex >= 0 ? (this._id + '-list' + selectedIndex) : null }
```

Comment:
is it valid to have aria-activedescendant with a blank value? You could mix in a conditional object

```ts
{ ...( isOpen && selectedIndex >= 0 ? { 'aria-activedescendant': ... } : null ) }
```